src/extensions/default/AutoUpdate/node/AutoUpdateDomain.js

```diff
@@ -87,12 +91,19 @@
      * @param {string} messageId - Message to be passed
      */
     function postMessageToBrackets(messageId) {
+        if(!requesters[currentRequester]) {
+            for (var key in requesters) {
+                currentRequester = key;
```

Comment:
why do we need the requesters list if we always send message only to the currentRequester ?